.gitignore

```diff
@@ -15,3 +15,5 @@ www/
 src/app/_environments/environment.ts
 
 .DS_Store
+
```

Comment:
We do want to keep changes to the package-lock.json file updated.

https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5